### ISSUE TITLE
[REQUEST] More optimizations for the cockpit screens refresh rate

### ISSUE THREAD

**Issue by PuL9** [flybywiresim/a32nx#1391]
**Is your feature request related to a problem? Please describe.**

Yes and no, will describe in next step

**Describe the feature you'd like to see implemented**

Lowering cockpit screens refresh rate is a proven way to reduce CPU bottlenecking your GPU, fps boost in general
I came up with an idea to have different refresh rates for different screens (also dynamic refresh rate)

**PFD**
When cruising at FL390 the screen is pretty much static, changes are only visible if you zoom in and look for speed indicator to move by a couple of pixels.
Idea: if there is not much happening, like speed and altitude is stable, lower it's refresh rate (to 5 fps?) and normal fps (based on what you have set in game settings) when climbing/descending

**MCDU**
Does not require more than 3-5 fps (1 fps will be annoying when typing) or dynamic like PFD

**ECAM lower**
Door and Cruise pages aren't showing critical information and can be lowered to 1-5 fps

**Other screens**

Chronometer display: should be fixed at 1 fps because it shows new information only once a second

Transponder (squawk, frequency displays) + Barometric pressure and the wide screen next to it (with autopilot speed etc, not sure how its called) are static 95% of the flight,  1 fps (or don't update it at all) if nothing is being changed, normal (as stated above in PFD idea) when you are changing stuff

PFD, ND on the copilot side: half from captain's side screens

Battery voltage displays: 1 fps or even lower (update once in 5 seconds?)

**References**

https://drive.google.com/file/d/1nFlewseVVfpEI2nIoMcOLKb2yCN3V6tn/view?usp=sharing

**Additional context**

i9 9900KF + RTX 3080, 2K Ultra settings, 50-60 fps inside cockpit with low screens refresh rate in game settings, 70-90fps in external view.
Turning copilot's ND+PFD off (brightness to 0) gives me 5-10 fps boost (just to know how huge the impact is)

Discord username same

**Comment by Benjozork** [comment]
Honestly, we'd rather improve the efficiency of our code than hack our way through the performance problems.

We are going to work on this, but keep in mind it's midterm season and a lot of people are inactive.

**Comment by 2hwk** [comment]
Duplicate of #1341

**Comment by MisterChocker** [comment]
Still duplicate.
